models/pp_pop_psc_delta.cpp

```diff
@@ -296,7 +296,7 @@ nest::pp_pop_psc_delta::calibrate()
       temp = 0;
     }
 
-    for ( int j = 0; j < V_.len_eta_; j++ )
+    for ( int j = 0; j < V_.len_eta_ - 1; j++ )
```

Comment:
This looks like a very brutal fix. I looked some more at the pp_pop_psc_delta code and noticed (a) that the code is in poor shape and (b) that it has carried since Nov 2016 a deprecation notice referring users to gif_pop_psc_exp. I would therefore suggest to remove the model entirely. @jougs @terhorstd What do you think?

Reply:
I suggest to remove the model in 3.2 and add a corresponding sentence to [the detailed release notes of 3.1](https://github.com/nest/nest-simulator/blob/master/doc/userdoc/release_notes/v3.1/index.rst).

Reply:
Just to clarify: please add the deprecation info for the model to the release notes in this PR, and open a new issue with the 3.2 milestone set about the removal of the model.

Reply:
I agree with @jougs. After some more code-staring, I now also understand why the `-1` is correct above. Even though we will remove the model in 3.2, could you add a comment above like
```python
# Set all except last element to zero.
```

Reply:
Given that the code is soon disappearing, I'd keep changes minimal, so your `-1` solution is fine. Just put this comment before the loop:
```python
# Set all except last state vector elements to zero, then fill last element with initial value
```

Reply:
Got it.